backend/core/src/listings/listings.controller.ts

```diff
@@ -35,13 +35,17 @@ import { ListingsRetrieveQueryParams } from "./dto/listings-retrieve-query-param
 import { ListingCreateValidationPipe } from "./validation-pipes/listing-create-validation-pipe"
 import { ListingUpdateValidationPipe } from "./validation-pipes/listing-update-validation-pipe"
 import { ListingLangCacheInterceptor } from "../shared/interceptors/listing-lang-cache.interceptor"
+import { ActivityLogInterceptor } from "../activity-log/interceptors/activity-log.interceptor"
+import { ActivityLogMetadata } from "../activity-log/decorators/activity-log-metadata.decorator"
 
 @Controller("listings")
 @ApiTags("listings")
 @ApiBearerAuth()
 @ResourceType("listing")
 @ApiExtraModels(ListingFilterParams)
 @UseGuards(OptionalAuthGuard, AuthzGuard)
+@ActivityLogMetadata([{ targetPropertyName: "status", propertyPath: "status" }])
```

Comment:
@seanmalbert Yes because "propertyPath" supports providing a dot delimited path e.g. "user.jurisdiction.name", so they could be different. What should be the metadata key then? I figured giving it a custom name is convenient, it could be the same but it does not have to.